editor/components/webgl/surface.ts

```diff
@@ -113,9 +113,7 @@ export class WebGLSurface implements AfterViewInit, OnDestroy {
     }
 
     ngOnDestroy(): void {
-        this._loop = () => {
-            this.gl = undefined;
-        };
+        this._loop = () => {};
```

Comment:
Maybe you can write `let break_loop = () => {} in this._loop = break_loop`. That would be clearer? Or maybe not.